### PR TITLE
Update lua-output.rst

### PR DESCRIPTION
**Pull request by salpagarov**
Log filename building fixed

Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
-
-
-

#suricata-verify-pr:
#suricata-verify-repo:
#suricata-verify-branch:
#suricata-update-pr:
#suricata-update-repo:
#suricata-update-branch:
#libhtp-pr:
#libhtp-repo:
#libhtp-branch:

### CODE REVIEW

**Comment by norg** [comment]
LGTM, not sure what to thinks about the coverage diff just for this small change

File: doc/userguide/output/lua-output.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ Example:
   end
 
   function setup (args)
-      filename = SCLogPath() .. "/" .. name
+      filename = SCLogPath() .. "example.log"
```
Comment (jasonish) [inline doc/userguide/output/lua-output.rst:37]: I don't think is is correct.  We have S-V tests that use the first form just fine.  If you leave a trailing '/' in your log directory configuration, then yes, `filename` will end up with a double `//` in it, but is this a problem?  If your log directory configuration does not have a trailing `/` in it (which is typical of a default install) then this new form will fail.

Comment (salpagarov) [reply doc/userguide/output/lua-output.rst:37]: ok
but what about _empty_ 'name' variable in this script?

Comment (jasonish) [reply doc/userguide/output/lua-output.rst:37]: > but what about _empty_ 'name' variable in this script?

Yeah, the probably should be fixed.

**Pull request closed without merging**